R/helpers_kintr_Increments.R

```diff
@@ -0,0 +1,318 @@
+#' @importFrom knitr knit_print
+NULL
+
+# Increments ----
+
+#' Render a IncrementsRelative Object
+#'
+#' @description `r lifecycle::badge("experimental")`
+#' @inherit knit_print.CohortSizeConst return
+#' @param ... passed to `knitr::kable`
```

Comment:
```suggestion
#' @param ... passed to [knitr::kable()] 
```
so the reader can click on the link

Reply:
Should this be [`knitr::kable`]?

Reply:
OK.